.github/workflows/integrate.yaml

```diff
@@ -100,7 +100,10 @@ jobs:
           charmcraft-channel: latest/candidate
 
       - run: |
```

Comment:
nit: let's add a name to this step so that the comment doesn't show up as the title of the step in the CI

Reply:
I put in a fix for this